tests/end_to_end/test_client_json.py

```diff
@@ -351,8 +353,42 @@ def testSearchVariantAnnotations(self):
 
     # def testSearchRnaQuantifications(self):  # TODO
 
-    # def testGetExpressionLevels(self): # TODO
-
-    # def testGetRnaQuantifications(self): # TODO
+    # TODO SqliteRnaQuantificationSet.getExpressionLevel is
+    # passing in an RnaQuantSet to the ExpressionLevel constructor
+    # instead of a RnaQuant
+    @unittest.skip("expression levels not initialized correctly")
```

Comment:
this looks like an error.  `getExpressionLevel` exists at both the rnaQuantSet and rnaQuant levels.  It should not be at the rnaQuantSet level